### ISSUE TITLE
unable to read arbitrary content from cookie - Invalid Number Exception

### ISSUE THREAD

**Issue by rogerthemoose** [reagent-project/reagent-utils#10]
I have a random token "5wcDN..." stored in a cookie but am unable to read it back as the reader attempts to parse it as a number and throws 'Invalid number format'

```
Uncaught Error: Invalid number format [5wcDNnJcfcNJiJQ495qDiKE7YyX0SRMYe2iU36eW2uLKKdn066toSIS7nrQNAWLg%2FpWV2dKWXnyrCylm]
    at Function.cljs.reader.reader_error.cljs$core$IFn$_invoke$arity$variadic (reader.cljs:71)
    at cljs$reader$reader_error (reader.cljs:69)
    at cljs$reader$read_number (reader.cljs:301)
    at cljs$reader$read (reader.cljs:465)
    at cljs$reader$read_string (reader.cljs:477)
    at reagent$cookies$read_value (cookies.cljs:23)
```

Why does the get function make use of 'read-string' to read the value from the cookie ? 

```
# taken from reagent.cookies starting line 21

(defn- read-value [v]
  (when v
    (reader/read-string v)))

(defn get
  "gets the value at the key, optional default when value is not found"
  [k & [default]]
  (or
    (->> (name k) (.get goog.net.cookies) read-value)
    default))
```

**Comment by yogthos** [comment]
The current implementation assumes EDN content in cookies. I'd be open for a PR to make this less restrictive. Perhaps, could add a new function like `get-raw` in order to maintain backwards compatibility.

**Comment by taylorwood** [comment]
I think this can be closed after #11.

**Comment by yogthos** [comment]
👍
